File: Iniciante/2061.py
lines1 = list(map(int, input().split()))

abas = lines1[0]
vezes = lines1[1]

for i in range(vezes):
  acao = str(input())

  if (acao == 'fechou'):
    abas -= 1
    abas += 2
  else:
    abas -= 1

print(abas)

File: Iniciante/2061.js
let lines1 = lines.shift().split(' ');

let abas = Number(lines1.shift());
let vezes = Number(lines1.shift());;

for (let i = 0; i < vezes; i++) {
  let acao = lines.shift();

  if (acao == "fechou") {
    abas--;
    abas += 2;
  } else {
    abas--;
  }
}

console.log(abas);
